index.html

```diff
@@ -0,0 +1,77 @@
+            </li>
+            <li>
+                <p><strong>1941</strong> - Tries to enroll in the military after the Pearl Harbor attack, but is rejected. Instead, the military asked his lab to work on waterproof glue, DDT to control malaria, disinfectants, and other applied science. </p>
+            </li>
+            <li>
+                <p><strong>1944</strong> - Rejects a 100% salary increase from Dupont, leaves behind his pregnant wife, and flies to Mexico to head a new plant pathology program. Over the next 16 years, his team breeds 6,000 different strains of disease resistent wheat - including different varieties for each major climate on Earth. </p>
+            </li>
+            <li>
+                <p><strong>1953</strong> - crosses a short, sturdy dwarf breed of wheat with a high-yeidling American breed, creating a strain that responds well to fertilizer. It goes on to provide 95% of Mexico's wheat. </p>
+            </li>
+            <li>
+                <p><strong>1962</strong>1 - Visits Delhi and brings his high-yielding strains of wheat to the Indian subcontinent in time to help mitigate mass starvation due to a rapidly expanding population </p>
+            </li>
+            <li>
+                <p><strong>2005</strong> - states "we will have to double the world food supply by 2050." Argues that genetically modified crops are the only way we can meet the demand, as we run out of arable land. Says that GM crops are not inherently dangerous because "we've been genetically modifying plants and animals for a long time. Long before we called it science, people were selecting the best breeds." </p>
+            </li>
+            <li>
+                <p><strong>2009</strong> - dies at the age of 95.</p>
+            </li>
+        </ul>
+        
+            <p>"Borlaug's life and achievement are testimony to the far-reaching contribution that one man's towering intellect, persistence and scientific vision can make to human peace and progress." </p>
+        
+        <div>
+            <p>-- Indian Prime Minister Manmohan Singh</p>
+        </div>
+
+    </article>
+    <p class="link"><strong>If you have time, you should read more about this incredible human being on his : </strong> <a href="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js." id="tribute-link"  target="_blank">Wikepdeia Page</a></p>
+
+    
+    <!-- <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js."></script> -->
+    <!-- <p>If you want to read more about these doctors, vist this : <a id="tribute-link" href="google.com" target="_blank">Wikepdeia Page</a></p>
+    -->
+    </div>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+</body>
```

Comment:
- [ ] Great job but be careful with blank lines and indentation.
- Do not add blank lines, spaces, or indentations without a reason at least for readability, add two spaces of indentation. do not use the tab key and add blank lines to separate large or logical code blocks.